### PR TITLE
Issue #2931932 by larowlan: Explore adding support for config entities

### PR DESCRIPTION
**Pull request by larowlan**
See https://www.drupal.org/project/workbench_access/issues/2931932

### CODE REVIEW

**Comment by agentrickard** [comment]
Not sure what the test error actually means here.

File: tests/src/Kernel/FilterAccessTest.php [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+use Drupal\Tests\workbench_access\Functional\WorkbenchAccessTestTrait;
+use Drupal\workbench_access\Entity\AccessScheme;
+use Drupal\workbench_access\WorkbenchAccessManagerInterface;
+
+/**
+ * Tests workbench_access integration with entity_test.
+ *
+ * @group workbench_access
+ */
+class EntityTestAccessTest extends KernelTestBase {
```
Comment (larowlan) [inline tests/src/Kernel/FilterAccessTest.php]: doh!

**Comment by agentrickard** [comment]
Closing / re-opening to trigger a Travis rebuild.

**Comment by agentrickard** [comment]
This is outdated and keeping failing. I'm going to pull it into a new branch for testing.

**Pull request closed without merging**